src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -4056,18 +4056,18 @@ void DynamicIntConvSurfaceClassification(EnergyPlusData &state, int const SurfNu
                 case DataZoneEquipment::ZoneEquipType::VentilatedSlab:
                 case DataZoneEquipment::ZoneEquipType::LowTemperatureRadiant: {
                     if (zoneEquipConfig.InFloorActiveElement) {
-                        for (int spaceNum : zone.spaceIndexes) {
-                            auto const &thisSpace = state.dataHeatBal->space(spaceNum);
+                        for (int spaceNumLoop : zone.spaceIndexes) {
+                            auto const &thisSpace = state.dataHeatBal->space(spaceNumLoop);
 
                             for (int SurfLoop = thisSpace.HTSurfaceFirst; SurfLoop <= thisSpace.HTSurfaceLast; ++SurfLoop) {
 
                                 if (!state.dataSurface->surfIntConv(SurfLoop).hasActiveInIt) continue;
-                                auto &surface = state.dataSurface->Surface(SurfLoop);
-                                if (surface.Class != SurfaceClass::Floor) continue;
+                                auto const &surfaceLoop = state.dataSurface->Surface(SurfLoop);
+                                if (surfaceLoop.Class != SurfaceClass::Floor) continue;
```

Comment:
[src/EnergyPlus/ConvectionCoefficients.cc:4065]:(style),[constVariable],Variable 'surface' can be declared as reference to const